lib/bootparams.js

```diff
@@ -430,6 +434,20 @@ function getBootParams(opts, callback) {
                     fArg.platforms = platforms;
                     cb();
                 });
+            },
+
+            function _getNodeConfig(fArg, cb) {
+                mod_node_config.getNodeConfig(opts, function getCfgCb(getErr) {
```

Comment:
The `getNodeConfig` should be returning the filepath to the node config file (rather than recreating it again below - actually see below if it's even used/needed).